applications/vanilla/views/categories/helper_functions.php

```diff
@@ -93,9 +93,15 @@ function getOptions($category) {
 endif;
 
 if (!function_exists('MostRecentString')):
-    function mostRecentString($row) {
-        if (!$row['LastTitle'])
+    function mostRecentString($row, $options = []) {
```

Comment:
What's the reason for the array cast here? Is there something out there known to be passing a second parameter to this function?

Reply:
There is not a known case, but PHP allows it.